accounting/entries.go

```diff
@@ -241,3 +241,52 @@ func invoiceEntry(invoice *lnrpc.Invoice, circularReceipt bool,
 		hash, preimage, note, false, convert,
 	)
 }
+
+// forwardTxid provides a best effort txid using incoming and outgoing channel
+// ID paired with timestamp in an effort to make txid unique per htlc forwarded.
+// This is not used as a reference because we could theoretically have duplicate
+// timestamps.
+func forwardTxid(chanIn, chanOut, unixTx uint64) string {
```

Comment:
nit/note: If we were to change this to `ts:in:out` then we'd have a natural "global" order which may or may not be useful.

Reply:
Also, should we consider padding the `chanIn/Out` and the `unixTx` to get an equal lenght tx id?